spikeinterface/sorters/pykilosort/pykilosort.py

```diff
@@ -97,5 +97,5 @@ def _run_from_folder(cls, output_folder, params, verbose):
 
     @classmethod
     def _get_result_from_folder(cls, output_folder):
-        raise NotImplementedError
         # return sorting
+        return read_kilosort(output_folder,*args, **kwargs) # Changed to use existing reader
```

Comment:
Hi @yonibrowning 

Note that `*args` and `**kwargs` are undefined within the scope of the function. Since the output of `pykilosort` should be the same as kilosort (phy format), you can do this (see https://github.com/SpikeInterface/spikeinterface/blob/master/spikeinterface/sorters/kilosortbase.py#L44-L51):

```
# at the top
from ..kilosortbase import KilosortBase

@classmethod
def _get_result_from_folder(cls, output_folder):
    return KilosortBase._get_result_from_folder(cls, output_folder)
```

Reply:
It looks like 
`return KilosortBase._get_result_from_folder(cls, output_folder)`
expects to find the spikeinterface**.json files in the output_folder directory as the pykilosort results.

I find 
return read_kilosort(output_folder/'output')
better handles the idiosyncrasies of the pykilosort format.